### PR TITLE
Ccw 4483 indent fix

### PR DESCRIPTION
**Pull request by chinnadhurai**
@adri-romsor
@nouiz
@mgermain

Comments:
Pls ignore pull request gh-4616.
The following functions are deprecated

toolbox.py / NodeFinder
utils.py / type_guard
cc.py / get_persistent_module_cache

This is a follow up of issue #4428 
Fixed all the pep-8 errors in /gof folder including unify.py

### CODE REVIEW

**Comment by nouiz** [comment]
travis have this error: 

/home/travis/build/Theano/Theano/theano/gof/graph.py:docstring of theano.gof.graph.variables_and_orphans:5: SEVERE: Unexpected section title.

**Comment by chinnadhurai** [comment]
Not able to figure out the reason for this error. Any idea?

**Comment by nouiz** [comment]
/home/travis/build/Theano/Theano/theano/gof/graph.py:docstring of theano.gof.graph.variables_and_orphans:5: SEVERE: Unexpected section title.

In the file theano.gof.graph.variables, in the function variables_and_orphans, the Parameter section of the docstring have wrong number of - under it.

File: theano/gof/cc.py [hunk 1]
```diff
@@ -1183,8 +1169,8 @@ def make_thunk(self, input_storage=None, output_storage=None,
             be allocated.
         storage_map: dict that map variables to storages.
             This is used when you need to customize the storage of
-            this thunk.
-
+            this thunk.keep_lock
+        WRITEME
```
Comment (mgermain) [inline theano/gof/cc.py]: This should be removed or written no ?

File: theano/gof/cc.py [hunk 2]
```diff
@@ -117,7 +115,12 @@ def failure_code(sub):
 def failure_code_init(sub):
     """
     Code for failure in the struct init.
-
+    Parameters:
+    --------------
```
Comment (mgermain) [inline theano/gof/cc.py]: You should always match the number of - with the letters above in docstring.

File: theano/gof/fg.py [hunk 3]
```diff
@@ -365,6 +362,7 @@ def __import_r__(self, variable, reason):
         """
         Import variables to this FunctionGraph and also their apply_node,
         if those nodes are not in this graph.
+        TO DO : explain the parameter "reason"
```
Comment (mgermain) [inline theano/gof/fg.py]: Should you do this TODO ?

Comment (lamblin) [inline theano/gof/fg.py]: `reason` is the name of the optimization or operation in progress.

**Comment by chinnadhurai** [comment]
Fixes done based on Fred's comments

**Comment by nouiz** [comment]
thanks. I think it is good.

**Pull request merged**